sql/moz-fx-data-marketing-prod/acoustic/contact_current_snapshot_v1/init.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS
```

Comment:
Do we actually need this given we have the schema and metadata?

Reply:
I think you're right about this. I'll remove that file.